crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -251,9 +251,8 @@ where
         // Pre-visit.
         match stmt {
             Stmt::Global(ast::StmtGlobal { names, range: _ }) => {
-                let ranges: Vec<TextRange> = identifier::names(stmt, self.locator).collect();
```

Comment:
Only semi-related (noticed during the refactor), but we no longer need to do this -- we have ranges on the names.